source/wrangling.md

```diff
@@ -541,41 +368,42 @@ Melting data from a wide to long data format.
 ```
 
 We can achieve this effect in Python using the `.melt` function from the `pandas` package.
-The `.melt` function combines columns, 
-and is usually used during tidying data 
-when we need to make the data frame longer and narrower. 
+We say that we "melt" (or "pivot") the wide table into a longer format.
```

Comment:
I would remove this new "We say that" sentence -- especially because "pivot" is actually a different function in python (the R naming is so much better in this case...pivot_wider and pivot_longer. Python has "melt" and "pivot").